viewer-prototype/src/browser/trace-server-status.ts

```diff
@@ -80,15 +100,34 @@ export class TraceServerConnectionStatusContribution extends DefaultFrontendAppl
 
     protected handleOnline(): void {
         this.toDisposeOnOnline.dispose();
+        this.statusBar.setElement(this.statusbarId, {
+            alignment: StatusBarAlignment.LEFT,
+            text: '$(fas fa-stop) Stop trace server',
```

Comment:
This is really great, except that it stopped my entire browser!! (or it crashed I'm not sure)

Reply:
Fixed